app/views/recipe_foods/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Ingredients for <%= @recipe.name %></h2>
+
+<table class="recipe-table">
+  <thead class="table-head">
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @recipe_foods.each do |recipe_food| %>
+      <tr>
+        <td><%= recipe_food.food.name %></td>
+        <td><%= recipe_food.quantity %></td>
+        <td><%= number_to_currency(recipe_food.total_price) %></td>
+      </tr>
+    <% end %>
+  </tbody>
```

Comment:
- [ ] Additionally, our recipe section needs a few adjustments to align more closely with the original wireframe. Let's fine-tune it for a consistent look! 📋✨

![image](https://github.com/hajnaloltyan/recipe-app/assets/98453979/bed05018-c0fd-4389-bd45-d38ece66fd20)

 #### The original wireframe

![image](https://github.com/hajnaloltyan/recipe-app/assets/98453979/6220b754-e983-4321-bddd-1d7a226cd28c)

- [ ]  After creating a shopping list, your app displays a routing error despite successfully generating the list. It'd be great to ensure that upon creating a shopping list, the app redirects to the actual shopping list page for a smoother user experience. 📝🛒

![image](https://github.com/hajnaloltyan/recipe-app/assets/98453979/8553d485-77fa-485a-9540-7e67d04e3754)